arangod/Aql/HashedCollectExecutor.cpp

```diff
@@ -191,6 +196,18 @@ void HashedCollectExecutor::writeCurrentGroupToOutput(
                            _lastInitializedInputRow, guard);
     }
   }
+  if (_infos.getCollectRegister().value() != RegisterId::maxRegisterId) {
+    auto& builder = *_currentGroup->second.second;
+    builder.close();
+
+    AqlValue val(std::move(*builder.steal()));
+    AqlValueGuard guard{val, true};
+    builder.clear();
+    _currentGroup->second.second.reset();
```

Comment:
NIT The reset will destroy the builder, so the `builder.clear()` is unnecessary.

Reply:
Done